test.properties

```diff
@@ -6972,160 +6972,154 @@
 
-    test.batch.run.property.query[functional-tomcat90-postgresql121-jdk8][publications-environments]=\
+    test.batch.run.property.query[functional-tomcat90-postgresql121-jdk8][publications-db]=\
         (database.types ~ postgresql) AND \
         (testray.main.component.name == "Publications")
 
-    test.batch.run.property.query[functional-tomcat90-sqlserver2019-jdk8][publications-environments]=\
+    test.batch.run.property.query[functional-tomcat90-sqlserver2019-jdk8][publications-db]=\
         (database.types ~ sqlserver) AND \
         (testray.main.component.name == "Publications")
 
-    test.batch.run.property.query[functional-weblogic*-mysql57-jdk8][publications-environments]=\
+    test.batch.run.property.query[functional-upgrade-tomcat90-db2111-jdk8][publications-db]=\
+        (database.types ~ db2) AND \
+        (testray.main.component.name == "Upgrades Publications")
+
+    test.batch.run.property.query[functional-upgrade-tomcat90-mariadb102-jdk8][publications-db]=\
+        (database.types ~ mariadb) AND \
+        (testray.main.component.name == "Upgrades Publications")
+
+    test.batch.run.property.query[functional-upgrade-tomcat90-oracle193-jdk8][publications-db]=\
+        (database.types ~ oracle) AND \
+        (testray.main.component.name == "Upgrades Publications")
+
+    test.batch.run.property.query[functional-upgrade-tomcat90-postgresql*-jdk8][publications-db]=\
+        (database.types ~ postgresql) AND \
+        (testray.main.component.name == "Upgrades Publications")
+
+    test.batch.run.property.query[functional-upgrade-tomcat90-sqlserver2017-jdk8][publications-db]=\
+        (database.types ~ sqlserver) AND \
+        (testray.main.component.name == "Upgrades Publications")
+
+    #
+    # Publications with Database Partition
+    #
+
+    database.partition.enabled[publications-db-partition]=true
+    test.batch.dist.app.servers[publications-db-partition]=tomcat
+
+    test.batch.names[publications-db-partition]=functional-tomcat90-mysql57-jdk8
+
+    test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][publications-db-partition]=\
+        ${test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][publications]}
+
+    #
+    # Publications with Different Application Servers
+    #
+
+    test.batch.class.names.includes[publications-as]=${test.batch.class.names.includes[publications]}
+    test.batch.dist.app.servers[publications-as]=${test.batch.dist.app.servers}
+
+    test.batch.names[publications-as]=\
+        functional-jboss74-mysql57-jdk8,\
+        functional-upgrade-jboss74-mysql57-jdk8,\
+        functional-upgrade-weblogic122-mysql57-jdk8,\
+        #functional-upgrade-weblogic141-mysql57-jdk8,\
+        functional-upgrade-websphere90-mysql57-jdk8,\
+        functional-weblogic141-mysql57-jdk8,\
+        functional-websphere90-mysql57-jdk8,\
+        functional-wildfly230-mysql57-jdk8,\
+        modules-integration-hypersonic20-jdk8
```

Comment:
`modules-integration-hypersonic20-jdk8` doesn't need to be included in this test suite since it sets a unique database